kairos-server/src/l1_sync/interval_trigger.rs

```diff
@@ -0,0 +1,20 @@
+use tokio::time::{self, Duration};
+
+use std::sync::Arc;
+
+use super::service::L1SyncService;
+
+pub async fn run(sync_service: Arc<L1SyncService>) {
+    let mut interval = time::interval(Duration::from_secs(30));
+
+    loop {
+        interval.tick().await;
+
+        tracing::debug!("Triggering periodic L1 sync");
+        let result = sync_service.trigger_sync().await;
+
+        if let Err(e) = result {
+            tracing::error!("Unable to trigger sync: {}", e);
+        }
```

Comment:
Nit: use `map_err`

Reply:
Done in https://github.com/cspr-rad/kairos/pull/96/commits/be86df8504c256394e59ca2ee20cca25424ee819.